CONTRIBUTING.md

```diff
@@ -99,4 +99,9 @@ Syntax:
 
 And in case we didn't emphasize it enough: we love tests!
 
+## Writing Tests
+
+* We do write tests for our warns and assertion messages. You may use the `assert.expectAssertion()` and `assert.expectWarning()` helpers for this. Note: Our Travis tests that will run on your PR run in the `production` envrionment, where warnings and assertions are stripped. If you're writing a test for these, you should use `testInDebug` helper rather than `test` to define your test.
+
```

Comment:
That's good also! Just in case you haven't thought already about that: I would suggest a link using the specific commit, and not `master`, so the link stays resilient to changes. You can achieve that by pressing `y` to get the canonical form: https://help.github.com/articles/getting-permanent-links-to-files/.